srcpkgs/StyLua/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'StyLua'
+pkgname=StyLua
+version=0.11.0
+revision=1
+build_style=cargo
+short_desc="Opinionated Lua code formatter"
+maintainer="Omar Zeghouani <omarzeghouanii@gmail.com>"
+license="MPL-2.0"
+homepage="https://github.com/JohnnyMorganz/StyLua"
+changelog="https://raw.githubusercontent.com/JohnnyMorganz/StyLua/master/CHANGELOG.md"
+distfiles="https://github.com/JohnnyMorganz/StyLua/archive/v${version}.tar.gz"
+checksum=39f0bad4cc175202eae2551e4ddaf3dd6a229943e8da3e462d4fa15a024fd0fa
+
+build_options="lua52 luau"
+build_options_default="lua52 luau"
+
+desc_option_lua52="Enable support for Lua 5.2 syntax"
+desc_option_luau="Enable support for Luau syntax"
+
+_features="$(vopt_if lua52 'lua52,')"
+_features+="$(vopt_if luau 'luau')"
+
+if [ "$_features" ]; then
+	echo $_features
```

Comment:
Do not echo anything on the top level (It is fine if you do it for debugging for yourself, but don't keep it in the final template)